### PR TITLE
feat: Import basic operator data

### PR DESCRIPTION
**Pull request by mathcolo**
Asana Task: [Import basic operator data into Orbit backend](https://app.asana.com/0/1206105669438487/1207392635878919/f)

- Oban worker to import personnel data every morning at 3:50AM.
- Add preferred first name to `employees`.

Checklist

- Tests:
  - `(x)` Has tests
  - `( )` Doesn't need tests
  - `( )` Tests deferred (with justification)
- Product/Design sign off:
  - `(x)` Okayed the plan for the feature (e.g. the design files, or the Asana task)
  - `( )` Reviewed the feature as implemented (e.g. on dev-green, or saw screenshots)
  - `( )` No review needed

### CODE REVIEW

File: test/orbit/import/personnel_test.exs [hunk 1]
```diff
@@ -0,0 +1,64 @@
+defmodule Orbit.Import.PersonnelTest do
```
Comment (skyqrose) [inline test/orbit/import/personnel_test.exs:1]: Would it be possible to also have a test for `download()` that loads the data from `demo.csv`, so the loading and parsing and oban worker is tested?

If testing the oban worker is hard, you might still be able to test the rest of the loading process by refactoring some logic out of `Orbit.Import.PersonnelWorker.perform` and into a function in `Orbit.Import.Personnel`.

File: test/orbit/import/personnel_test.exs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      "CUSTOM_FIELD3" => "Area",
+      "CUSTOM_VALUE3" => "115"
+    }
+
+    Personnel.import_rows([row], MapSet.new(["114"]))
+
+    assert nil == Repo.one(from(e in Employee, where: e.first_name == "Christopher"))
+  end
+
+  test "properly formats fields" do
```
Comment (skyqrose) [inline test/orbit/import/personnel_test.exs:43]: You could add `"WORK_EMAIL_ADDRESS" => ""` and `email: nil` to also test that `empty_to_nil` works correctly.

File: config/config.exs [hunk 3]
```diff
@@ -17,10 +17,15 @@ config :orbit, OrbitWeb.Endpoint,
   pubsub_server: Orbit.PubSub,
   live_view: [signing_salt: "5ANP5/by"]
 
+# Personnel import
+config :orbit, Orbit.Import.Personnel, pathname: nil
```
[the remark this answers is not in the document]
Comment (mathcolo) [reply config/config.exs:21]: I added a `perform()` test, so this is no longer the case.

File: test/orbit/import/personnel_test.exs (hunk 1 above)
Comment (mathcolo) [reply test/orbit/import/personnel_test.exs:1]: Done!

**Review by skyqrose — APPROVED** [review]
Approved (once terraform is ready with the env vars)

**Pull request merged**